contrib/BLAKE3/build_rust_lib.cmake

```diff
@@ -0,0 +1,106 @@
+function(build_cargo target_name project_dir)
```

Comment:
It seems [corrosion](https://github.com/corrosion-rs/corrosion) might be able to provide the same functionality out of the box. Check it out: https://github.com/ClickHouse/ClickHouse/pull/41225